components/primitives/Button/Button.tsx

```diff
@@ -0,0 +1,50 @@
+import { cn } from "@/lib/utils/cn";
```

Comment:
BLOCKING

Kindly rename the component file to a lowercase format.

Going from `Button.tsx` to `button.tsx`

Reply:
A little curious to as why the lowercase convention for Components🤔

Is this to help migitate differences between how different operating systems resolve filenames, or is it purely an opinionated approach? Thoughts🤔